### DISCUSSION TITLE
Install guide for Updated Alpha?

### DISCUSSION THREAD

**Discussion by netstacks** [serverlessworkflow/synapse#370] [General]
I know there is a big change in the Serverless workflow DSL and the Alpha version is in the repo. Is there a way to install and use this software for the public yet? I seem to not be able to build the docker-compose. I do not use .NET framework so there is many files I do not understand in the Repo

**Comment by cdavernas** [comment]
@viperbmw You can do the following, as explained in the wiki, which has already been updated for `1.0.0-alpha1`:

1. **Clone the Synapse Repository:**

    ```bash
    git clone https://github.com/serverlessworkflow/synapse.git
    ```
    
   **1.1. Checkout to branch  `1.0.0-alpha1`:**

    ```bash
    git fetch origin
    git switch '1.0.0-alpha1'
    ```

2. **Navigate to the Docker Compose directory:**

    ```bash
    cd synapse/deployments/docker-compose
    ```

3. **Build the Docker images:**

    ```bash
    docker-compose -f docker-compose.build.yml build
    ```

4. **Start the Synapse services:**

    ```bash
    docker-compose -f docker-compose.build.yml up
    ```
    
You can then navigate to [http;//localhost:8080](http;//localhost:8080) to reach the dashboard. To access it, you'll need to supply an access token when requested. 
Those are configured in the `config/tokens.yaml` file, which defines a default one: `h0v0fixhkg9z4bgtxele7892x9sjtw7o`.

I hope this helps!

**Comment by cdavernas** [comment]
@viperbmw Any progress? Could you get started? Don't hesitate to reach out if need be!
